client/rest-high-level/src/test/java/org/elasticsearch/client/RequestTests.java

```diff
@@ -77,20 +79,42 @@
 
 public class RequestTests extends ESTestCase {
 
+    public void testConstructor() throws Exception {
```

Comment:
can we also test the visibility of the class and its constructor by checking their modifiers?